pagopa/io-app
# feat: [IOBP-790] Add local toggle to switch on/off the new "Scan" section

## Short description
This PR adds a local toggle to switch on/off the new tab architecture with the Scan section and the new "Settings" section into header actions. Since from the `2.65.0` the new payment section screen is activated through a remote feature flag, in this PR I provided to delete also the local FF of that section.

## List of changes proposed in this pull request
- Removed the previous local feature flag of new payments screen;
- Added a new FF `isNewScanSectionLocallyEnabledSelector`;
- Added a toggle inside the profile developer page to enable/disable the new scan and profile section;
- Removed from the .env variables the `SHOW_BARCODE_SCAN_SECTION` since it isn't used anymore with the introduction of this local feature flag switch;
- Removed the global variable `showBarcodeScanSection` and replaced with the selector above;

## How to test
- Open the profile section
- Scroll down until the "Nuova sezione scansiona" switch is visible
- Toggle it and you should be able to see real-time the new architecture

**NB:** When you toggle it on or off, the navigator changes under the hood so it will redirect you in a different screen 


## Preview

https://github.com/user-attachments/assets/479a5156-4e45-4f3b-9d85-205a233f54d5
